### ISSUE TITLE
domain of morph:baseType

### ISSUE THREAD

**Issue by matteo-pellegrini** [ontolex/morph#23]
In the current draft, there is a datatype property morph:baseType whose definition states that it is "used for coindexing a base form, an inflection rule and the forms generated by the rule from the respective base in cases in which the inflectional paradigm of a single lexical entry involves different bases, e.g., stems."

Hence, currently the domain is required to be either ontolex:Form or morph:InflectionRule.

However, it is quite commonly recognised in the morphological literature that also word formation rules can be applied to bases that are not the citation form of the lexeme (see e.g. Aronoff 1994 for Latin)
For instance, in Latin, deverbal agent and action nouns in -_or_ and -_io_ are formed on a stem allomorph different than the present stem that is displayed in the citation form of verbs - namely, what Aronoff 1994 calls the "third stem".
E.g., for the verb meaning 'to write'
- citation form of the verb: _scribo_ (displaying present stem _scrib_-)
- supine: _scriptum_ (displaying third stem _script_-)
- agent and action nouns: _script-or_, _script-io_.

To account for similar cases, I wonder whether it might be reasonable to change the definition and domain of morph:baseType so as to include word formation rules - i.e., with domain either ontolex:Form or **morph:Rule**.

In this way, the Latin example can be modelled easily, and similarly to how it is shown in the current example for inflection, i.e. as follows:

    :scribo_entry a ontolex:LexicalEntry ;
	    ontolex:canonicalForm :scribo_form ;
	    morph:baseForm :scriptum_form .
    
    :scribo_form a ontolex:Form ;
	    ontolex:writtenRep 'scribo' .
	    morph:baseType "PresentStem" .
    
    :scriptum_form a ontolex:Form ;
	    ontolex:writtenRep 'scriptum' ;
	    morph:baseType "ThirdStem" .
    
    agent_rule a morph:WordFormationRule ;
	    morph:baseType "ThirdStem" ;
	    morph:replacement [
	    :source "um$"
	    :target "or"]
    
    action_rule a morph:WordFormationRule ;
	    morph:baseType "ThirdStem" ;
	    morph:replacement [
	    :source "um$"
	    :target "io"
	    ] .

Otherwise, how can such a state of affairs be handled? Should the generation of derivatives always be based on the canonical form?

**Comment by max-ionov** [comment]
Yes, absolutely, this should be the case. I am surprised it is restricted to morph:InflectionRule only, it clearly should be ontolex:Form or morp:Rule, like you suggest

**Comment by jmccrae** [comment]
@max-ionov @matteo-pellegrini could you implement this change?

**Comment by matteo-pellegrini** [comment]
I just pushed a proposal where I did some minimal changes to definitions and examples of baseType in the section on inflection, and added the derivational example proposed here in the section on the word formation. Feel free to change/mark as closed if you think it works.

**Comment by max-ionov** [comment]
That is actually a perfect solution to one of the long-standing problems in the draft, I am closing this
